ReleaseChecklist.md

```diff
@@ -79,6 +80,7 @@ At least a day before the release:
  - [ ] Update the version and the hash (``sha256sum solidity_$VERSION.tar.gz``) in the [``solidity`` formula in Homebrew core repository](https://github.com/Homebrew/homebrew-core/blob/master/Formula/s/solidity.rb).
 
 ### Docker
+ - [ ] Make sure `docker-buildx` is installed.
```

Comment:
```suggestion
 - [ ] Make sure ``docker-buildx`` is installed.
```

Though I'd also be fine with refactoring this file in a separate PR to switch to single backticks and end this insanity :)

Reply:
Yeah let's end it :) I'll prepare a PR tomorrow (or rather: later today)

Reply:
We could also do that with the `CHANGELOG`. I would not to as far as to reformat the older entries (`git blame` is really useful there and that would make it more annoying to use) but I think it would be fine to just switch to the new style now since we just released and the changelog for the new release is empty. We could also remove the annoying space before the bullet point, which people keep removing (understandably, since it's a weird way to do it).

 What do you think?

Reply:
Yup, absolutely for it. It is a bit annoying that we get an inconsistent style with older entries but on the other hand it is a clean cut and more sensible from now on. :)